packages/core/src/Parser.ts

```diff
@@ -18,12 +19,49 @@ interface ParsingContext {
     format?: FormatType[];
     vDocument: VDocument;
 }
+export const defaultNodes = [CharNode, LineBreakNode, RangeNode, SimpleElementNode];
 
 export class Parser {
+    _customVNodes: Set<typeof VNode> = new Set();
+    constructor(replaceDefaultNodes?: Array<typeof VNode>) {
```

Comment:
We replace the list, so this implies that if the person wants to add a custom node, he loses the other functionalities (ex adds ImageNode but loses CharNode if it's missing in the list)?